privacyidea/lib/smsprovider/SMSProvider.py

```diff
@@ -70,6 +71,10 @@ def __str__(self):
 
 class ISMSProvider(object):
     """ the SMS Provider Interface - BaseClass """
+
+    REGEXP_description = "Regular expression to modify the phone number to make it compatible with provider." \
```

Comment:
You should add the translation marker here.

Reply:
https://github.com/privacyidea/privacyidea/pull/3597/commits/77a9d792a5a2fa7d0a136c0e2e20cf3a8de76585